example/linux/Makefile

```diff
@@ -89,7 +85,11 @@ LDFLAGS=-L$(BUNDLE_LIB_DIR) \
 # Targets
 
 .PHONY: all
-all: $(BIN_OUT) bundle
+all: sync $(BIN_OUT) bundle
+
+.PHONY: sync
```

Comment:
Add a comment here saying that it's phony because we don't know the inputs, so we trust the Flutter tooling to run only when necessary.